frontend/public/module/k8s/k8s-actions.ts

```diff
@@ -37,6 +39,11 @@ const actions = {
   [types.errored]: action_(types.errored),
   [types.modifyObject]: action_(types.modifyObject),
 
+  watchAPIServices: () => dispatch => {
+    dispatch({type: types.getResourcesInFlight});
+    dispatch(actions.watchK8sList(makeReduxID(APIServiceModel, {}), {}, APIServiceModel, actions.getResources));
```

Comment:
I'm not sure if "regular" users will have RBAC permissions to list this cluster-wide resource.

Reply:
Not in an out of the box OpenShift install :/

> apiservices.apiregistration.k8s.io is forbidden: User "user@example.com" cannot list apiservices.apiregistration.k8s.io at the cluster scope: no RBAC policy matched

Reply:
It's weird that API discovery currently works for a "regular" user using `/api/v1` and `/apis/` which shows the exact same information as `/apis/apiregistration.k8s.io/v1/apiservices`.